### ISSUE TITLE
Update code to account for standardized RENT endpoints

### ISSUE THREAD

**Issue by gabrielesh** [hackoregon/housing-frontend#33]
I'm cross-posting @jaymcgrath 's note in the housing-frontend slack, to make sure that frontend gets updated in response:
Hey front end peeps.. just a heads up, RENT endpoint names are now standardized..

[6:38] 
`nh_id` is now `NP_ID` and `nh_name` is now `neighborhood` to match the other endpoints

[6:39] 
also added fields to the `prodvscost` endpoint for @esme3d 's weighted averages.. the endpoint looks like this now:

[6:39] 
 ```"year": "2014",
    "NP_ID": 1,
    "neighborhood": "122-Division",
    "single_unit_growth": 2113.523694,
    "multi_unit_growth": 3220.607533,
    "home_price_growth": 125998.9316,
    "rent_growth": 32956.23342,
    "msa_growth": 1.468620461,
    "weighting_factor": 8283```

[6:40] 
Also,`homeownership` was added as a `housing_size` to the `rent` endpoint, and `overall` was removed. Unfortunately, we only have `homeownership` data for 2015

**Comment by DingoEatingFuzz** [comment]
The changes to the rent endpoint were addressed by #28 

Are the changes to `prodvscost` related to #24?

**Comment by gabrielesh** [comment]
Oh great! sorry to muddy the waters @DingoEatingFuzz ! I'll close this issue.
The changes to prodvscost are so that in the future we can do a live interactive version of this info rather than the static charts that John is working on for #24
